torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -372,6 +376,17 @@ def async_wrong_type():
 def async_add(to, x, y):
     return rpc.rpc_async(to, torch.add, args=(x, y))
 
+
+def slow_add(x, y):
+    time.sleep(1)
+    return torch.add(x, y)
```

Comment:
do we need to add a test with CUDA ops as well?

Reply:
Yes, that's a good point.